### ISSUE TITLE
@elementsByXXX do not return elements

### ISSUE THREAD

**Issue by matti** [sebv/node-wd-sync#6]
All of "@elementsByXXX" methods just return something like this (based on the number of matches). I've no idea how to fix this :worried: 

```
[ { value: '0',
    browser: 
     { sessionID: '61988c81-afcb-405a-8671-4a6f68025eda',
       username: undefined,
       accessKey: undefined,
       basePath: '/wd/hub',
       https: false,
       options: [Object],
       defaultCapabilities: [Object] } },
  { value: '1',
    browser: 
     { sessionID: '61988c81-afcb-405a-8671-4a6f68025eda',
       username: undefined,
       accessKey: undefined,
       basePath: '/wd/hub',
       https: false,
       options: [Object],
       defaultCapabilities: [Object] } },
  { value: '2',
    browser: 
     { sessionID: '61988c81-afcb-405a-8671-4a6f68025eda',
       username: undefined,


```

**Comment by sebv** [comment]
You can do it like this.
`
 var queryField = browser.elementByName('q');
 queryField.type(queryField);
`
Or pass it as a parameter like in the example.

**Comment by alindsay55661** [comment]
I think what he is getting at is that when you make a call to return a single element you get a nicely wrapped object with methods like `click`. However when you get back multiple elements all you have is a `value` property. What can be done with this? How can we get back a list of wrapped objects like the call for a single element?

**Comment by matti** [comment]
Yep, 

That was what I was trying to say. I spent some time with the code, but couldnt come up with anything.

**Comment by sebv** [comment]
Looks like It is already wrapped.

Below is an example: 

``` coffeescript
wdSync = require 'wd-sync' 

{browser, sync} = wdSync.remote()

sync ->        
  console.log "server status:", @status()
  @init browserName:'firefox'

  @get "http://google.com"
  console.log @title()          

  els = @elementsByTagName 'div'

  # console.log els

  for el in els
    el.click (err) ->
      console.log("clicked", err);

  @quit()  
```

Let me know if you still have issue, and provide a code sample and I'll reopen.

**Comment by sebv** [comment]
Should be fixed in 1.2.0. Sorry it took me some time to understand what the issue was . Next time it would be best to provide some code samples.
